examples/processbenchmarks.jl

```diff
@@ -0,0 +1,206 @@
+status_map = Dict(
+    "Optimal" => "co",
+    "KilledTime" => "tl",
+    "TimeLimit" => "tl",
+    "SlowProgress" => "sp",
+    "NumericalFailure" => "er",
+    "SkippedSolveCheck" => "sk",
+    "SetupModelKilledTime" => "tl",
+    "SolveCheckKilledTime" => "tl",
+    "SetupModelKilledMemory" => "rl",
+    "SolveCheckKilledMemory" => "rl",
+    # TODO remove if not needed
+    "SetupModelCaughtError" => "er",
+    "SolveCheckCaughtError" => "er",
+    )
+
+residual_tol_satisfied(a, tol = 1e-6) = (all(isfinite, a) && maximum(a) < tol)
+relative_tol_satisfied(a::T, b::T, tol::T = 1e-5) where {T <: Real} = (abs(a - b) / (1 + max(abs(a), abs(b))) < tol)
+
+function make_all_df()
+    all_df = DataFrame(CSV.File(bench_file))
+    transform!(
+        all_df,
+        [:inst_set, :solver] => ((x, y) -> x .* "_" .* y) => :inst_solver,
+        [:x_viol, :y_viol, :z_viol, :rel_obj_diff] => ByRow((res...) -> residual_tol_satisfied(res)) => :converged,
+        :status => ByRow(x -> status_map[x]) => :status,
+        )
+    return all_df
+end
+
+ex_wide_file(ex_name::String) = joinpath(output_folder, ex_name * "_wide.csv")
+
+function make_wide_csv(all_df, ex_name, ex_params)
+    @info("making wide csv for $ex_name")
+    ex_df = all_df[all_df.example .== ex_name, :]
+
+    # make columns out of tuple
+    inst_keys = ex_params[1]
+    for (name, pos) in zip(inst_keys, ex_params[2])
+        transform!(ex_df, :inst_data => ByRow(x -> eval(Meta.parse(x))[pos]) => name)
+    end
+
+    # check objectives if solver claims optimality
+    for group_df in groupby(ex_df, inst_keys)
+        # check all pairs of converged results
+        co_idxs = findall(group_df[:status] .== "co")
+        if length(co_idxs) >= 2
+            for i in eachindex(co_idxs)
+                first_optval = group_df[co_idxs[i], :prim_obj]
+                other_optvals = group_df[co_idxs[Not(i)], :prim_obj]
+                if !all(relative_tol_satisfied.(other_optvals, first_optval))
+                    println("objective values of: $(ex_name) $(group_df[:inst_data][1]) do not agree")
+                end
+            end
+        end
+    end
+
+    # TODO check that ext npq agrees for each formulation-instance
+    unstacked_dims = [
+        unstack(ex_df, inst_keys, :inst_set, v, renamecols = x -> Symbol(v, :_, x))
+        for v in [:n, :p, :q]
+        ]
+    unstacked_res = [
+        unstack(ex_df, inst_keys, :inst_solver, v, renamecols = x -> Symbol(v, :_, x))
+        for v in [:status, :converged, :iters, :solve_time]
+        ]
+    ex_df_wide = join(unstacked_dims..., unstacked_res..., on = inst_keys)
+    CSV.write(ex_wide_file(ex_name), ex_df_wide)
+
+    return ex_df_wide
+end
+
+process_entry(::Missing) = "\$\\ast\$"
+process_entry(::Missing, ::Missing) = "sk"
+process_entry(x::Int) = (isnan(x) ? "\$\\ast\$" : string(x))
+function process_entry(x::Float64)
+    isnan(x) && return "\$\\ast\$"
+    @assert x > 0
+    if x < 1
```

Comment:
if 0.99 <= x <= 1 this prints as `.0`, which can be fixed by making the if condition if x < 0.99 (which should get rounded to 1.0 and not start with a `.`)